weblate/static/daterangepicker-dark.css

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © Michal Čihař <michal@weblate.org>
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+@media (prefers-color-scheme: dark) {
```

Comment:
This will not work for manually selected dark theme, please place this CSS into `weblate/static/style-dark.css` which is loaded in that case.

Reply:
Deleted `daterangepicker-dark.css` and moved it contents to `style-dark.css`.

Reply:
The file seems still present (the content was added to `style-dark.css`, but the file was kept here).